autogluon/training/docker/0.4.0/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,131 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+ARG PYTHON_VERSION=3.8.12
+ARG PYTHON_VER=3.8
+ARG FROM_IMAGE=rapidsai/rapidsai-core
+
+FROM ${FROM_IMAGE}:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG CUDA_VER
+
+ARG OPENSSL_VERSION=1.1.1l
+ARG AUTOGLUON_VERSION=0.4.0
+
+ARG MXNET_VER=1.9.*
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    PYTHONIOENCODING=UTF-8 \
+    LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    MXNET_CUDNN_LIB_CHECKING=0 \
+    SAGEMAKER_TRAINING_MODULE=sagemaker_mxnet_container.training:main \
+    LD_LIBRARY_PATH=/opt/conda/envs/rapids/lib:/usr/local/lib:$LD_LIBRARY_PATH \
+    PATH=/opt/conda/envs/rapids/bin:/usr/local/bin:$PATH
+
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+    libgl1-mesa-glx \
+    ffmpeg \
+    libsm6 \
+    libxext6 \
+    emacs \
+    vim \
+ && apt-get autoremove -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm -rf /tmp/*
+
+# install OpenSSL
+RUN wget -q -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
+ && tar -xzf openssl-${OPENSSL_VERSION}.tar.gz \
+ && cd openssl-${OPENSSL_VERSION} \
+ && ./config && make -j $(nproc) && make install \
+ && ldconfig \
+ && cd .. && rm -rf openssl-* \
+ && rmdir /usr/local/ssl/certs \
+ && ln -s /etc/ssl/certs /usr/local/ssl/certs
+
+ARG CUDNN_VERSION=8.*
```

Comment:
Please define this at the top of the dockerfile, so that it can be easily found and modified if needed. Perhaps next to `CUDA_VER`?

Reply:
done